src/components/MessagesList/MessagesGroup/Message/MessagePart/ReactionsList.vue

```diff
@@ -0,0 +1,199 @@
+<!--
+  - @copyright Copyright (c) 2024 Dorra Jaouad <dorra.jaoued7@gmail.com>
+  -
+  - @author Dorra Jaouad <dorra.jaoued7@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<!-- all reactions -->
+<template>
+	<NcModal size="small"
+		:container="container"
+		@close="closeModal">
+		<div class="reactions__modal">
+			<h2>
+				{{ t('spreed', 'Reactions') }}
+			</h2>
+			<template v-if="Object.keys(reactionsOverview).length > 0">
+				<div class="reactions-list__navigation">
+					<NcButton v-for="reaction in reactionsMenu"
+						:key="reaction"
+						:class="{'active' : reactionFilter === reaction}"
+						type="tertiary"
+						@click="handleTabClick(reaction)">
+						{{ reaction }}
+						<span v-if="reaction === 'All'">({{ reactionsOverview[reaction].length }})</span>
+						<span v-else>{{ reactionsOverview[reaction].length }}</span>
```

Comment:
```suggestion
						{{ (reaction === 'all' ? t('spreed', 'All') : reaction) + ' ' + reactionsOverview[reaction].length }}
```

Reply:
It was split for the sake of parenthesis.

Reply:
Ah, didn't notice them 🙈
But I think it also looks nice

Reply:
@nextcloud/designers